plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -35,12 +35,16 @@ public static function load() {
 	 * @return mixed $settings
 	 */
 	public static function add_marketplace_settings( $settings ) {
+		$auth_user_data  = WC_Helper_Options::get( 'auth_user_data' );
```

Comment:
It would be nice to specify a default value, I reckon.

```suggestion
		$auth_user_data  = WC_Helper_Options::get( 'auth_user_data', array() );
```